AutoCollection/ServerRequestParser.ts

```diff
@@ -110,6 +106,10 @@ class ServerRequestParser extends RequestParser {
         return this.requestId;
     }
 
+    public getCorrelationContext() {
+        return this.correlationContext;
```

Comment:
Right now this getter is how we get access to the value to put it in the custom properties; all the header parsing happens in this class.